external-crates/move/crates/move-analyzer/trace-adapter/src/adapter.ts

```diff
@@ -431,46 +431,13 @@ export class MoveDebugSession extends LoggingDebugSession {
         this.sendResponse(response);
     }
 
-    protected stepBackRequest(
```

Comment:
Removal of code related to stepping through the trace backwards is unrelated but it was time to do it as it has been slowly rotting. Will bring it back when the time is right